### PR TITLE
Added signbit() to cpp docs

### LINKED ISSUE #989: [Term Entry] C++ Math Functions: `signbit()`
### Reason/inspiration (optional)

Under `cpp/concepts/math-functions/terms/`, create a new term entry for the `signbit()` function.

- In the terms/ folder, make a new folder called `signbit`
- Inside the `signbit` folder, create a new markdown file called `signbit.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Let's include an "Example" section with a \```cpp code snippet.
- Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [X] Issue is assigned

### PR DESCRIPTION
**Pull request by AravindShounik**
### Description

Closes #989 
Created folder named signbit at path: cpp/concepts/math-functions/terms/
Created a file named signbit.md, which includes all relevant data for a Term Entry Issue.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by AravindShounik** [comment]
Could you please review the changes

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:3]:
```suggestion
Description: 'Returns true if the floating point argument is negative and false if floating point argument is positive.'
```

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:14]:
```suggestion
The **`signbit()`** function returns true if the sign of the argument is negative and false if its sign is positive.
```

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:35]:
```suggestion
    double a=9.0, b=0.0;
```

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:41]:
```suggestion
        std::cout << "c tends to positive infinity";
```

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:45]:
```suggestion
        std::cout << "c tends to negative infinity";
```

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:67]:
```suggestion
    std::cout << "n is negative";
```

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:71]:
```suggestion
    std::cout << "n is positive";
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@AravindShounik, I did a first review and left a few comments.

**Comment by AravindShounik** [comment]
I have updated the file according to your suggestions. @SSwiniarski. Please look into them

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:14]:
```suggestion
The **`signbit()`** function returns true if the sign of the argument is negative and false if its sign is positive.
```

**Review by SSwiniarski — APPROVED** [review]
@AravindShounik one extra space needs to be trimmed, but otherwise looks good for a second review.

**Comment by AravindShounik** [comment]
Thank you for taking your time to review my PR @SSwiniarski . Eagerly waiting for the next review.

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:59]:
```suggestion
int main()
{
```

**Comment by Dusch4593** [comment]
@AravindShounik let us know if you need any help with implementing the latest round of feedback. Today is the last day to have your merged PR count for Hacktoberfest!

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:40]:
```suggestion
    if (std::signbit(c) == false)
```

**Review by yangc95 — APPROVED** [review]

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:24]:
```suggestion
Argument `n` must be of type `double`/`float`/`long double`/`int`, and the return value will be of that type. This function also detects the signs of zeroes, infinities, and `NaN`s.
```

File: content/cpp/concepts/math-functions/terms/signbit/signbit.md
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/signbit/signbit.md:54]:
```suggestion
The example below outputs a string about whether the integer is positive or negative:
```

**Comment by yangc95** [comment]
@AravindShounik very cool, this is the live link: https://www.codecademy.com/resources/docs/c-sharp/variables

Screen Shot 2022-11-04 at 12 52 47 AM

cc: @SSwiniarski

**Pull request merged**
